ovve_ui/ovve_ui.py

```diff
@@ -275,20 +275,23 @@ def updateGraphs(self) -> None:
         self.flow_graph_line.setData(self.flow_data[:self.graph_ptr+1])
         self.flow_graph_cache_line.setData(self.flow_data[self.graph_ptr+2:])
         self.flow_graph_cache_line.setPos(self.graph_ptr+2, 0)
+        self.flow_graph_line.show()
```

Comment:
Is there any cost associated with calling show every time even when it's not hidden?

Reply:
It likely just sets a flag on the widget telling it to be visible.  I didn't want to have a special case for the first time we enter the function which would probably be about the same time to check the condition.  You could time it and see but I bet it's negligible.

Reply:
Okay. Seems good to me then!